src/providerhub/azext_providerhub/generated/_params.py

```diff
@@ -87,18 +88,22 @@ def load_arguments(self, _):
         c.argument('provider_namespace', type=str,
                    help='The name of the resource provider hosted within ProviderHub.')
         c.argument('rollout_name', type=str, help='The rollout name.')
-        c.argument('row2_wait_duration', type=str, help='The wait duration before the rollout '
-                   'begins in rest of the world two.')
-        c.argument('skip_regions', action=AddSkipRegions,
+        c.argument('rest_of_the_world_group_two', options_list=['--rest-of-the-world-group-two', '--row2'], action=AddDefaultRolloutROW2,
+                   nargs='+', help='DefaultRollout rest of the world group two.')
+        c.argument('canary', action=AddCanaryRegion,
                    nargs='*', help='The canary regions to skip.')
+        c.argument('row2_wait_duration',
+                   type=str, help='The rest of the world group two wait duration.')
+        c.argument('skip_regions',
+                   action=AddSkipRegions, nargs='+', help='The canary skip regions.')
 
     with self.argument_context('providerhub default-rollout update') as c:
         c.argument('provider_namespace', type=str,
                    help='The name of the resource provider hosted within ProviderHub.')
         c.argument('rollout_name', type=str, help='The rollout name.')
-        c.argument('row2_wait_duration', type=str, help='The wait duration before the rollout begins '
-                   'in rest of the world two.')
-        c.argument('skip_regions', action=AddSkipRegions,
+        c.argument('rest_of_the_world_group_two', options_list=['--rest-of-the-world-group-two', '--row2'], action=AddDefaultRolloutROW2,
+                   nargs='+', help='DefaultRollout rest of the world group two.')
+        c.argument('canary', action=AddCanaryRegion,
```

Comment:
There arguments still exist.

Reply:
Would like to keep these to be consistent with CustomRollouts, both of which are part of the rollout API group. As noted above, I had added support for the previous version to avoid breaking change for the DefaultRollouts commands. Please let me know if that'd work.